### DISCUSSION TITLE
Welcome to linter Discussions!

### DISCUSSION THREAD

**Discussion by mihir-bombay-studio** [dear-digital/linter#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by abhishekjani08** [comment]
Sir, as stated in the ReadMe file, a mentorship session will be held every Saturday at 11:00 a.m. I attempted to join the earlier meet link, but no one accepted me. Could you please tell me what time I should join?

**Reply by mihir-bombay-studio** [reply]
Hi @abhishekjani08 , sorry my bad I forgot to add the meet to my calender. Let me know when you'd like to have a meet tomorrow or anytime in the next week.

**Comment by tanujbordikar** [comment]
hey @mihir-bombay-studio , i visited the updated code and i am found this bug in which when i backspace the { and ", it keep occuring again and again. So its better to remove that feature.
I have searched a npm package called as monaco-editor. In which auto-spacing and auto brackets issue will be resolved. 
Any suggestions??

https://github.com/dear-digital/linter/assets/91179998/fcd0dc69-2ebf-477f-b4e8-2084154742f9

**Reply by mihir-bombay-studio** [reply]
Thanks for finding and pointing the bug @tanujbordikar please open an issue with a short description of whichever solution you finally decide would be better for the repository. To fix the bug or to add a new package to replace it and why.

**Reply by tanujbordikar** [reply]
@mihir-bombay-studio okay sure i will open one and provide proper description of how to handle that!

**Comment by anishdalvi** [comment]
## **Important Announcement  📢** 

Hello everyone,

I wanted to let you know that the issues you have raised until now will be assigned to you after next week Tuesday. In the meantime, please feel free to continue raising any new issues or discussing topics related to the repository. **Your active participation is greatly appreciated.**

Thank you for your contributions!
-- By our Project Maintainer Mihir Sir @mihir-bombay-studio
